apps/academy-wep/components/ready-to-earn.tsx

```
@@ -0,0 +1,112 @@
+'use client'
+
+import { Button } from '@/components/ui/button'
+import { Award } from 'lucide-react'
+import { useRouter } from 'next/navigation'
+interface ReadyToEarnProps {
+	badges: Array<{ id: string; name: string; earned: boolean }>
+}
+'use client'
+
+import { Button } from '@/components/ui/button'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Button'. Consider to delete it or rename it.

'Button' is defined here:


(lint/suspicious/noRedeclare)